man/systemd.netdev.xml

```diff
@@ -443,6 +443,18 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
```

Comment:
Also, could you fix the commit message of the third commit? `egress` -> `ingress`.